src/components/SubmissionForm/SubmissionForm.js

```diff
@@ -91,7 +92,7 @@ const SubmissionForm = () => {
   
     return (
     <section className="form-container">
-      <form onSubmit={event => handleSubmit(event)}>
+      <form>
```

Comment:
Tried to make a new submission and it looks like it isn't being posted, but noticed the handler is removed.

Reply:
Walked through this fix in person. It's resolved 🙂